vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/facade/ZephyrScaleFacade.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.zephyr.facade;
+
+import java.io.IOException;
+import java.util.Optional;
+import java.util.OptionalInt;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Configuration;
+import org.vividus.jira.JiraClient;
+import org.vividus.jira.JiraClientProvider;
+import org.vividus.jira.JiraConfigurationException;
+import org.vividus.jira.JiraFacade;
+import org.vividus.jira.model.Project;
+import org.vividus.util.json.JsonPathUtils;
+import org.vividus.zephyr.configuration.ZephyrConfiguration;
+import org.vividus.zephyr.configuration.ZephyrExporterConfiguration;
+import org.vividus.zephyr.configuration.ZephyrExporterProperties;
+
+@Configuration
+@ConditionalOnProperty(value = "zephyr.exporter.api-type", havingValue = "SCALE")
+public class ZephyrScaleFacade implements IZephyrFacade
+{
+    private static final String REST_ATM_ENDPOINT = "/rest/atm/1.0";
+    private static final String TEST_RESULT_ENDPOINT_FORMAT = "/testrun/%s/testcase/%s/testresult";
+
+    private final JiraFacade jiraFacade;
+    private final JiraClientProvider jiraClientProvider;
+    private final ZephyrExporterConfiguration zephyrExporterConfiguration;
+    private final ZephyrExporterProperties zephyrExporterProperties;
+    private ZephyrConfiguration zephyrConfiguration;
+
+    public ZephyrScaleFacade(JiraFacade jiraFacade, JiraClientProvider jiraClientProvider,
+                             ZephyrExporterConfiguration zephyrExporterConfiguration,
+                             ZephyrExporterProperties zephyrExporterProperties)
+    {
+        this.jiraFacade = jiraFacade;
+        this.jiraClientProvider = jiraClientProvider;
+        this.zephyrExporterConfiguration = zephyrExporterConfiguration;
+        this.zephyrExporterProperties = zephyrExporterProperties;
+    }
+
+    @Override
+    public Integer createExecution(String execution) throws IOException, JiraConfigurationException
+    {
+        String testCaseUrl = String.format(TEST_RESULT_ENDPOINT_FORMAT,
+                zephyrConfiguration.getCycleId(),
+                execution);
+        String responseBody = getJiraClient().executePost(
+                REST_ATM_ENDPOINT + testCaseUrl,
+                "{}");
```

Comment:
Why are you sending an empty body here?

Reply:
@VerkhovtsovPavel  could you please elaborate

Reply:
Body contains additional test case statuses and parameters, but at this step test a case only added to a test run.